current-reviews-data.ipynb

```diff
@@ -897,17 +952,378 @@
+       "      <th>2023Q3</th>\n",
+       "      <td>1</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th rowspan=\"2\" valign=\"top\">sneakers-the-rat</th>\n",
+       "      <th>2023Q4</th>\n",
+       "      <td>1</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2024Q2</th>\n",
+       "      <td>1</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>xmnlab</th>\n",
+       "      <th>2021Q4</th>\n",
+       "      <td>1</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>yeelauren</th>\n",
+       "      <th>2023Q3</th>\n",
+       "      <td>1</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>zeitsperre</th>\n",
+       "      <th>2024Q1</th>\n",
+       "      <td>1</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                                            n_edits\n",
+       "editor                          created_at         \n",
+       "Batalex                         2024Q1            1\n",
+       "cmarmo                          2024Q2            1\n",
+       "ctb                             2024Q1            1\n",
+       "dhomeier                        2024Q1            1\n",
+       "hamogu                          2024Q2            2\n",
+       "isabelizimm                     2024Q2            1\n",
+       "lwasser                         2019Q1            1\n",
+       "                                2019Q4            1\n",
+       "snacktavish Emily Jane McTavish 2023Q3            1\n",
+       "sneakers-the-rat                2023Q4            1\n",
+       "                                2024Q2            1\n",
+       "xmnlab                          2021Q4            1\n",
+       "yeelauren                       2023Q3            1\n",
+       "zeitsperre                      2024Q1            1"
+      ]
+     },
+     "execution_count": 13,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "n_edits = count_edits_by_quarter(reviews_df)[\"n_edits\"].to_frame()\n",
+    "for editor in ignore_editors:\n",
+    "    n_edits.drop(editor, inplace=True)\n",
+    "n_edits"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2024-07-14T23:34:19.359304Z",
+     "iopub.status.busy": "2024-07-14T23:34:19.359109Z",
+     "iopub.status.idle": "2024-07-14T23:34:19.364809Z",
+     "shell.execute_reply": "2024-07-14T23:34:19.364050Z",
+     "shell.execute_reply.started": "2024-07-14T23:34:19.359285Z"
+    },
+    "tags": [
+     "remove-input"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# Get axes min and max values so all plots have the same scale\n",
+    "\n",
+    "# Get vmin and vmax for the dates\n",
+    "min_date = reviews_df.created_at.min().to_period(\"Q\").start_time\n",
```

Comment:
Let's filter only on the currently open reviews and reviews with editors that are not in the ignore list. 


```python
ignore_editors = ["TBD", "lwasser", "xmnlab"]

current_reviews = edits[~edits['editor'].isin(ignore_editors)]
min_date = current_reviews.Date.min().to_period("Q").start_time
```
NOTE: let's add lwasser and xmnlab to the ignore list for these plots! 

then we can grab the min time from the list of current reviews! and the plots will look nicer !